uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java

```diff
@@ -0,0 +1,39 @@
+package com.microsoft.identity.client.ui.automation.utils;
+
+import androidx.test.uiautomator.UiDevice;
+
+import org.junit.Assert;
+
+import java.io.IOException;
+
+import static androidx.test.platform.app.InstrumentationRegistry.getInstrumentation;
+
+public class AdbShellUtils {
+
+    private static void executeShellCommand(final String command) {
+        UiDevice mDevice = UiDevice.getInstance(getInstrumentation());
+        try {
+            mDevice.executeShellCommand(command);
+        } catch (IOException e) {
+            Assert.fail(e.getMessage());
+        }
+    }
+
+    /**
+     * Remove the supplied package name from the device
+     *
+     * @param packageName the package name to remove
+     */
+    public static void removePackage(final String packageName) {
```

Comment:
Lots of places without any nullability annots -- these would be good to have. Will just call out once here

Reply:
Addressed